File: java-poc/src/main/java/io/craigmiller160/querydslpoc/java/data/entity/Position.java
package io.craigmiller160.querydslpoc.java.data.entity;

import java.util.UUID;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name = "positions")
public class Position {
  @Id private UUID id;
  private String name;
}

File: kotlin-poc/src/main/kotlin/io/craigmiller160/querydslpoc/kotlin/data/entity/Position.kt
package io.craigmiller160.querydslpoc.kotlin.data.entity

import java.util.UUID
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "positions")
class Position {
  @Id lateinit var id: UUID
  lateinit var name: String
}
